components/CollectiveNavbarActionsMenu.js

```diff
@@ -96,7 +96,7 @@ const CollectiveNavbarActionsMenu = ({
   }
 
   return (
-    <Container>
+    <Container display="flex" alignItems="center" flex={1} mx={2}>
```

Comment:
Since there's only one child, does `display="flex" alignItems="center"` have any effect?

Reply:
Yes, it was sitting at the top of its container and out of alignment with "Admin panel" otherwise.